rebar.config

```diff
@@ -19,7 +19,7 @@
 ]}.
 
 {deps, [
-    {quantile_estimator, "1.0.2"}
+    {ddskerl, "0.2.1"}
```

Comment:
Is `ddskerl` battle-tested? I saw it mentioned in the Erlang forums, but maybe it could be an "alternative" in this library and when it's fully battle-tested made the default?

Reply:
The point of the PR is to change the algorithm, and the algorithm is implemented in quantile_estimator/ddskerl. It's not just switching a dependency for another.

Perhaps we can require Nelson to run this PR in production for a while before merging if you're worried about the scale of the change.

Reply:
> The DDSketch algorithm is surely battle-tested

I didn't mean the algo, but the implementation 😄 I'm sure it's Ok, I just wanted to check what steps had been taken previously to ensure said Ok-ness (as @lhoguin points out: "if you're worried about the scale of the change.")

> I can add a note to the PR for people to see and choose if they want to try it out and come back to you in a few weeks when it is prod-tested though 👌🏽

This works for me.

Reply:
Done 👌🏽